lang/python39/Portfile

```diff
@@ -242,6 +242,12 @@ variant lto description {enable Link-Time Optimization} {
     configure.args-append   --with-lto
 }
 
+# Fixes the return type of `ttyname_r`
+# See https://github.com/python/cpython/issues/127614
+platform darwin 8 {
+    configure.cflags-append -D__DARWIN_UNIX03=1
```

Comment:
Should this be using `configure.cppflags`?

Reply:
No, the relevant code is in C not C++

Reply:
`CPPFLAGS` is flags for the C preprocessor. C++ flags go in `CXXFLAGS`.

Reply:
Ah, didn't know that, thank you! Done